build/Makefile

```diff
@@ -26,6 +26,22 @@ clean: clean-core-artifacts clean-readmes clean-json-content clean-xml-content c
 # Dependencies
 #
 
+
+
+
+OSCAL_CLI_VERSION:=1.0.3
+OSCAL_CLI_BIN:=oscal-cli
+OSCAL_CLI_INSTALL_URL:=https://repo1.maven.org/maven2/gov/nist/secauto/oscal/tools/oscal-cli/cli-core/$(OSCAL_CLI_VERSION)/cli-core-$(OSCAL_CLI_VERSION)-oscal-cli.zip
+OSCAL_CLI_INSTALL_PATH:=./oscal-cli
+OSCAL_CLI_PATH:=$(shell which $(OSCAL_CLI_BIN) 2>/dev/null || echo $(OSCAL_CLI_INSTALL_PATH))
```

Comment:
OSCAL_CLI_PATH in environments like mine will result in the full path to oscal-cli  because I have it installed ( aka /usr/local/bin/oscal-cli). It might be important to check if the already installed oscal-cli is the expected (latest) version.